platform/core/src/core.rs

```diff
@@ -65,7 +65,7 @@ pub fn bootstrap(
         return Err(CoreFailure::core_already_bootstrapped());
     }
     let preferences = Arc::new(Preferences {
-        store: Box::new(DefaultKeyValueStore::new(fs_root + PREFERENCES_DB)),
+        store: Box::new(DefaultKeyValueStore::new(fs_root + "/" + PREFERENCES_DB)),
```

Comment:
would make sense to have a helper function here that adds the '/' for sure so we avoid future issues in case we forget it ?